components/benefit_expansion.js

```diff
@@ -72,12 +72,28 @@ export class BenefitExpansion extends Component {
       childBenefits,
       "family"
     );
+    const reduxState = this.props.reduxState;
+    const profileFilters = JSON.parse(
+      JSON.stringify(getProfileFilters(reduxState, this.props))
+    );
 
     let otherBenefits = t("benefits_b.eligible_open_veteran", {
       x: benefitName
     });
 
-    return (
+    return profileFilters["statusAndVitals"] === "deceased" ? (
```

Comment:
I agree with @brdunfield that there's a simpler way to do this. We just need to prevent the service member list from rendering if they are deceased. If you look at the `ChildBenefitList` component, it doesn't render if it is passed an empty list as the benefits prop. So you could set the `vetServBenefits = []` if `reduxState.statusAndVitals === "deceased"`.